spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 
 RSpec.configure do |config|
   config.include Spec::Support::AutomationHelper
+  config.filter_run_excluding(:clean_yaml) if Psych.libyaml_version[1] > 1 # 0.1.x only
```

Comment:
Can we also add in a check for CI here?  We should _always_ run the spec on CI.

Reply:
Done!